resque-job_history.gemspec

```diff
@@ -2,13 +2,10 @@
 
 $LOAD_PATH.push File.expand_path("../lib", __FILE__)
 
-# Maintain your gem's version:
-require "resque/plugins/version"
```

Comment:
When attempting to run the specs, I was getting failures that this file could not be required and it wasn't being used in specs or gem build so I simply removed it.